Extension/package.json

```diff
@@ -38,7 +38,26 @@
     "Linters"
```

Comment:
The file is rewritten when the extension activates. Please only check in changes you have made to the package.json, and not the extension's rewritten content.